tests/helper/sampler_test.cpp

```diff
@@ -0,0 +1,75 @@
+// Unless explicitly stated otherwise all files in this repository are
```

Comment:
The scope doesn't seem to be tested here..

Reply:
Done